defs/scenarios/storage/ceph/osd_slow_ops.yaml

```diff
@@ -0,0 +1,35 @@
+checks:
+  osd_slow_ops:
+    input:
+      path: var/log/ceph/ceph*.log
+    expr: '^([\d-]+)T([\d:]+\.\d)\S+ .+ slow ops, oldest one blocked .+'
+  ceph_interfaces_have_errors:
+    requires:
+      property: hotsos.core.plugins.storage.ceph.CephChecksBase.has_interface_errors
+conclusions:
+  cause-unknown:
+    priority: 1
+    decision: osd_slow_ops
+    raises:
+      type: hotsos.core.issues.CephOSDError
+      message: >-
+        Cluster is experiencing slow ops.
+        Causes include network issues (could do connectivity checks via ping,
+        netstat/ip for packet drops), disk problems (check smartctl, sar,
+        dmesg for errors), or OSDs are too busy/hung. Also check if PGs are
+        in good state and can serve I/O (e.g. no inactive PGs).
+        Ensure 'ceph osd require-osd-release <release>' is set correctly.
```

Comment:
For this last point we could as for hotsos to be run on a ceph-mon so that https://github.com/canonical/hotsos/blob/main/defs/scenarios/storage/ceph/required_osd_release_mismatch.yaml gets run to check if it is a problem.